.env.development

```diff
@@ -7,7 +7,7 @@ LOGOUT_URL='http://localhost:18000/login'
 LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
 LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
 LOGO_WHITE_URL=https://edx-cdn.org/v3/default/logo-white.svg
-LOGO_POWERED_BY_OPEN_EDX_URL_SVG=https://edx-cdn.org/v3/stage/open-edx-tag.svg
+LOGO_POWERED_BY_OPEN_EDX_URL_SVG=''
```

Comment:
are you deploying the development package ?

Reply:
No, but is a way to also make it work in local dev.